transact/alchemy_simulateExecutionBundle.yaml

```diff
@@ -591,3 +592,11 @@ components:
             - type: string
         indexed:
           type: boolean
+
+    revertMsg:
```

Comment:
Let's capitalize this to RevertMessage for consistency with other components

Reply:
captitalized!